### PR TITLE
Relax otel process propagator vsn

### PR DESCRIPTION
**Pull request by bryannaegele**
The pinning of this dependency prevents updating the otel dataloader instrumentation library.

https://github.com/open-telemetry/opentelemetry-erlang-contrib/pull/284

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -63,7 +63,7 @@ defmodule Dataloader.Mixfile do
     [
       {:telemetry, "~> 1.0"},
       {:ecto, ">= 3.4.3 and < 4.0.0", optional: true},
-      {:opentelemetry_process_propagator, "~> 0.2.1", optional: true},
+      {:opentelemetry_process_propagator, "~> 0.3", optional: true},
```
Comment (benwilson512) [inline mix.exs:66]:
```suggestion
      {:opentelemetry_process_propagator, "~> 0.3 or ~> 0.2.1", optional: true},
```

**Comment by aspett** [comment]
@benwilson512 Do you have a moment to release this change by any chance, please? 🙏

**Comment by aspett** [comment]
🥇 That's speed! Legend, thank you.

**Comment by yordis** [comment]
@benwilson512 is there any opportunity to release a new version to get this change?

**Pull request merged**